src/lwc/gsAdminSetup/gsAdminSetup.js

```diff
@@ -0,0 +1,25 @@
+import { LightningElement, track } from 'lwc';
+
+import gsAdminSetupTitle from '@salesforce/label/c.gsAdminSetupTitle'
+
+import exploreNPSP from './exploreNPSP';
+import makeItYours from './makeItYours';
+import onboardUsers from './onboardUsers';
+
+export default class gsAdminSetup extends LightningElement {
+
+    @track checklists = [
+        { id: 1, ...exploreNPSP },
+        { id: 2, ...makeItYours },
+        { id: 3, ...onboardUsers },
```

Comment:
The problem with this approach is that in the future if a new element has to be added it is a little work to do:

1. Create a new component with the right parameters
2. Add it here.

I was thinking in Having a unique component like: gsChecklistItem

` export default class gsAdminSetup extends LightningElement { 
    @api title;
    @api desc;
}`

In this component(or an independent file) having a json like this one
`checklists = [
    {
        id: 1,
       title: <<label to section name>>,
       checklist_items: [{
           each item to be defined later on
      }]
    }
}`

And in this component, you build the instances of gsChecklistSection(which knows how to render itself).
This way if we want to change the content in the future(add a section for example) we need to just modify a single json